src/de.hpi.swa.graal.squeak/src/de/hpi/swa/graal/squeak/nodes/DispatchEagerlyNode.java

```diff
@@ -25,6 +25,7 @@
 @NodeInfo(cost = NodeCost.NONE)
 public abstract class DispatchEagerlyNode extends AbstractNodeWithCode {
     protected static final int INLINE_CACHE_SIZE = 6;
+    protected static final boolean TRUE = true;
```

Comment:
This is no longer necessary since [Truffle `19.3`](https://github.com/oracle/graal/blob/094363c11f3cf4882e0f90fc1788d19ded9f703d/truffle/CHANGELOG.md#version-1930). I'll take care of it :)